backend/config.py

```diff
@@ -1,6 +1,9 @@
 from diart import PipelineConfig
 from enum import Enum
 
+TRANSCRIPTION_DEVICE = "cuda"  # use 'cpu' if it doesn't work
+COMPUTE_TYPE = "int8_float16"  # use float32 with cpu
```

Comment:
Maybe I'm missing something, but the comment seems off. Float 32 though using `int8_float16`

Reply:
If one is using "cpu" as their transcription device, it should be float32. float16 wouldn't make that much of a difference even if supported.